mwibutsa/rich-form
# Upload images and store them to firebase

#### Description of changes made

> This PR adds functionality to upload images from

- File input
- Drag & Drop
- Clipboard

#### Type of change


#### How Has This Been Tested?


#### How should this be manually tested?

- After cloning the [repo](https://github.com/mwibutsa/rich-form.git)
- Check out this branch by `ft-connect-firebase`
- Run `yarn or npm install` to install the project dependencies
- Run `yarn start` to start the app and try copy & pasting images, drag and drop or click to upload them.

#### Checklist


#### GitHub issue your changes fix

#### Screenshots

   
![image](https://user-images.githubusercontent.com/36665761/112058901-667ee400-8b18-11eb-8ae9-6c1d586320fb.png)

![image](https://user-images.githubusercontent.com/36665761/112059102-aa71e900-8b18-11eb-9843-bea6a48fe8e9.png)
